common/incomingmessage.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package common
```

Comment:
I dont understand the reasoning for this file, can we not just import these functions from Offchain labs code?